devops/aws-codebuild/Jenkinsfile.cd

```diff
@@ -52,10 +51,11 @@ pipelineWrapper({
                 }
 
                 stage('Get last revision') {
-                    lastRevision = evernymRepo.getLastRevision {
+                    lastRevision = sovrinRepo.getLastRevision {
```

Comment:
This API doesn't exist, originally it was a part of EvernymRepo class which was renamed to NexusRepo.